data/XML/vocations.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <vocations>
-	<vocation id="0" clientid="0" name="None" description="none" gaincap="10" gainhp="5" gainmana="5" gainhpticks="6" gainhpamount="1" gainmanaticks="6" gainmanaamount="1" manamultiplier="4.0" attackspeed="2000" basespeed="220" soulmax="100" gainsoulticks="120" allowPvp="0" fromvoc="0">
```

Comment:
What does this mean? It isn't clear what changes between 0 and 1

Reply:
Isn't the ability to use magic shield guarded by the Lua spell script? This seems unnecessary - in client the magic shield bar won't show until magic shield is in use anyway.

Reply:
> What does this mean? It isn't clear what changes between 0 and 1

only mages should display the bar, and that is where this vocations.xml change comes in place

Reply:
So it should be a boolean (yes or no)

Reply:
> So it should be a boolean (yes or no)

we use 1/0 on almost every xml file (as you can see, even on allowPvp="0" on this line)

this can be changed in another PR as I'm just rebasing this commit, not fixing things, since the author is not active